contracts/VToken.sol

```diff
@@ -1349,6 +1367,55 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         emit NewMarketInterestRateModel(oldInterestRateModel, newInterestRateModel);
     }
 
+    /**
+     * @notice accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
```

Comment:
```suggestion
     * @notice Accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
```

Reply:
Besides params, natspec is rendered in the docs as if it is a complete sentence so the first letter should be capitalized.

Params are rendered as a table

Reply:
Actually in this and other comments we capitalize param so we should do that here for consistency